index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio | Edwin Gichuhi</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <a href="#">My Logo</a>
+                <span><img src="./assets/icons/Union.png" alt="menu"></span>
```

Comment:
- [ ]  it would be better if you change this to the `<button>` tag cause it's a button that will improve the SEO and readability